polymetis/polymetis/conf/robot_client/allegro_hardware.yaml

```diff
@@ -0,0 +1,96 @@
+defaults:
+  - robot_client/executable_cfg/velocity_filter: filters/butterworth_filters
+
+hz: 300
+use_real_time: false
+exec: allegro_hand_client
+
+control_port: 50051
+control_ip: 0.0.0.0
+
+robot_client:
+  _target_: polymetis.robot_client.executable_robot_client.ExecutableRobotClient
+  use_real_time: ${use_real_time}
+  metadata_cfg:
+    _target_: polymetis.robot_client.metadata.RobotClientMetadata
+    default_Kq: [ 3, 6, 3, 3, 6, 3, 3,3, 6, 3, 3, 3, 6, 3, 3, 3]
+    default_Kqd: [.1, .1, .06, .06, .06, .06, .06, .06, .06, .06, .06, .06, .06, .06, .06, .06]
+    default_Kx: [ 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
+    default_Kxd: [ 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
+    hz: ${hz}
+    robot_model_cfg: ${robot_model}
+  executable_cfg:
+    can_bus: PCI
+    control_ip: ${ip}
+    control_port: ${port}
+    readonly: false
+    mock: false
+    hz: ${hz}
+    exec: ${exec}
+    robot_client_metadata_path: ???
+    velocity_filter:
+      A: [ 1.000000000000000,  -1.561018075800718,   0.641351538057563 ]
+      B: [ 0.0200833655642112,   0.0401667311284225,   0.0200833655642112 ]
+      dt: .003
+    limits:
+      # bounding box of the workspace
+      cartesian_pos_upper:
+        - 1.0
+        - 0.4
+        - 1.0
+      cartesian_pos_lower:
+        - 0.1
+        - -0.4
+        - -0.05
+
+      # the remaining limits are set to the original franka limits minus a margin
+      joint_pos_upper: #margin: 0.1 rad
+        - 2.80
+        - 1.66
+        - 2.80
+        - -0.17
+        - 2.80
+        - 3.65
+        - 2.80
+      joint_pos_lower: #margin: 0.1 rad
+        - -2.80
+        - -1.66
+        - -2.80
+        - -2.97
+        - -2.80
+        - 0.08
+        - -2.80
+      joint_vel: #margin: 0.1 rad/s
+        - 2.075
+        - 2.075
+        - 2.075
+        - 2.075
+        - 2.51
+        - 2.51
+        - 2.51
+      elbow_vel: 2.075      #margin: 0.1 rad/s
+      joint_torques:  #margin: 1N for first 4 joints, 0.5N for last 3 joints
+        - 86.0
+        - 86.0
+        - 86.0
+        - 86.0
+        - 11.5
+        - 11.5
+        - 11.5
+
+    collision_behavior:
+      lower_torque: [20.0, 20.0, 20.0, 20.0, 20.0, 20.0, 20.0]
+      upper_torque: [20.0, 20.0, 20.0, 20.0, 20.0, 20.0, 20.0]
+      lower_force: [20.0, 20.0, 20.0, 20.0, 20.0, 20.0]
+      upper_force: [20.0, 20.0, 20.0, 20.0, 20.0, 20.0]
+
+    safety_controller:
+      is_active: true
+      margins: # margin from hard safety limits at which safety controllers start to kick in
+        cartesian_pos: 0.05
+        joint_pos: 0.2
+        joint_vel: 0.5
+      stiffness:
+        cartesian_pos: 200.0
+        joint_pos: 50.0
+        joint_vel: 20.0
```

Comment:
CMIIW, but since these fields aren't being used within the allegro client, there shouldn't be a need to include them in the config specific to allegro, right?

Reply:
Right - I'll delete the unused ones.